remoteit/desktop
# [DESK-501] Refactor registration screens to add help messaging

### Description


Provide improved device set up user experience with help.

### Checklist




### Test plan



1. Sign in to an unregistered system
2. Go to settings > device setup screen
3. Register your device
4. See a full screen loading page
5. See a full screen success page with help link
6. Be able to close the screen and see the device setup page
7. Device name should be in the title position with the delete icon at the top right

### Ticket



<https://remoteit.atlassian.net/browse/DESK-501>
